dashboard/src/main/webapp/src/pages/serverstatus/ExcludeNodeList.vue

```diff
@@ -17,21 +17,46 @@
 
 <template>
   <div>
-    <el-table
-      :data="pageData.tableData"
-      height="550"
-      style="width: 100%"
-      :default-sort="sortColumn"
-      @sort-change="sortChangeEvent"
-    >
-      <el-table-column prop="id" label="ExcludeNodeId" min-width="180" :sortable="true" />
-    </el-table>
+    <div style="text-align: right">
+      <el-button type="primary" @click="dialogFormVisible = true"> Add Node </el-button>
+    </div>
+    <div>
+      <el-table
+        :data="pageData.tableData"
+        height="550"
+        style="width: 100%"
+        :default-sort="sortColumn"
+        @sort-change="sortChangeEvent"
+      >
+        <el-table-column prop="id" label="ExcludeNodeId" min-width="180" :sortable="true" />
+      </el-table>
+      <el-dialog v-model="dialogFormVisible" title="Please enter Server:" class="dialog-wrapper">
```

Comment:
```suggestion
      <el-dialog v-model="dialogFormVisible" title="Please enter the exclude server id list" class="dialog-wrapper">
```

Reply:
All the questions you raised have been revised, please have a look at it.@xianjingfeng

Reply:
Please enter the server id list to be excluded